/**:
  ros__parameters:
    class_names: ["couch", "dining table", "chair", "potted plant", "bench"]